components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/persistence/JDBCSubscriberPersistenceManager.java

```diff
@@ -94,6 +94,8 @@ public void addSubscriber(PublisherDataHolder holder) throws EntitlementExceptio
         if (subscriberId == null) {
             throw new EntitlementException(ERROR_SUBSCRIBER_ID_NULL);
         }
+        PublisherPropertyDTO[] updatedPropertyDTOs = encryptSecretProperties(holder.getPropertyDTOs());
```

Comment:
Shall we change the variable name to signify that secret properties are encrypted?
eg: `propertyDTOsWithEncryptedSecrets`

Reply:
Addressed with https://github.com/wso2/carbon-identity-framework/compare/c5a3740b79a667bc27ae44ecf587dcbf130d8001..4ae683dccb165baec41091f765cd202f4fbe4fa9